testsuite/tests/typing-layouts-bits16/unboxed_bits16s.ml

```diff
@@ -39,7 +39,7 @@ end
 
 let to_binary_string x =
   String.init 16 (fun i ->
-    if Int16.(equal (logand x (shift_left (Stdlib_beta.Int16.of_int 1) (16 - i - 1))) (Stdlib_beta.Int16.of_int 0))
+    if Int16.(equal (logand x (shift_left (Stdlib_stable.Int16.of_int 1) (16 - i - 1))) (Stdlib_stable.Int16.of_int 0))
```

Comment:
I should have caught it the first time around, I guess, but I think all the `Stdlib_stable`s here are unnecessary becuse the file defines aliases to `Int16` and `Int16_u` at the top.  The tests for the pre-existing layouts just use the short names, I think.

(And similar comment for the equivalent files in bits8 / untagged-immediate)